app/src/main/java/me/hufman/androidautoidrive/music/controllers/SpotifyAppController.kt

```diff
@@ -311,10 +317,13 @@ class SpotifyAppController(context: Context, val remote: SpotifyAppRemote, val w
 			val item = recentlyPlayed?.items?.get(0)
 			if (item != null) {
 				remote.imagesApi.getImage(item.imageUri, Image.Dimension.THUMBNAIL).setResultCallback { coverArt ->
-					queueMetadata = QueueMetadata(item.title, item.subtitle, queueItems, coverArt, item.uri)
+					queueMetadata = QueueMetadata(queueTitle, item.subtitle, queueItems, coverArt, item.uri)
+
+					GlobalScope.launch(defaultDispatcher) {
+						val coverArtBase64 = Base64.encodeToString(Utils.compressBitmapJpg(coverArt, 100), Base64.NO_WRAP)
```

Comment:
A quality of ~85 is acceptable and saves a bit of space

Reply:
Good point, updated the quality to 85